src/python/pants/jvm/resolve/coursier_setup.py

```diff
@@ -75,6 +75,20 @@
 )
 
 
+POST_PROCESS_COURSIER_STDERR_SCRIPT = textwrap.dedent(
```

Comment:
Can you move the comment referring to https://github.com/pantsbuild/pants/issues/13942 here?